src/Athens-Examples/AthensSurfaceExamples.class.st

```diff
@@ -389,7 +392,7 @@ AthensSurfaceExamples class >> example6point1 [
 
 		can drawShape: (0@0 corner: 50@50) ].
 
-	Display getCanvas drawImage: surf asForm at: 0@0
+	surf asForm asMorph openInWindow
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?